src/main/java/io/javalin/core/JavalinConfig.java

```diff
@@ -105,6 +108,44 @@ public void enableRouteOverview(@NotNull String path, @NotNull Set<Role> permitt
         inner.routeOverview = new RouteOverviewConfig(path, permittedRoles);
     }
 
+    /**
```

Comment:
Could we pull these out into a separate config class? Or just have one methods that takes `OpenApiOptions`? Having six methods for one plugin seems a bit excessive.

Reply:
I moved all the overloads to the `OpenApiOptions`